cpp-ch/local-engine/Functions/SparkFunctionCheckDecimalOverflow.cpp

```diff
@@ -146,17 +156,20 @@ class FunctionCheckDecimalOverflow : public IFunction
     }
 
 private:
-    template <typename T, typename ToDataType>
+    template <typename FromDataType, typename ToDataType, typename ColVecType>
```

Comment:
ColVecType is not necessary. It could be deduced from `FromDataType`; 

`FromDataType -> FromFieldType -> ColumnVectorOrDecimal<FromFieldType>`